lib/models/index.js

```
@@ -24,16 +24,28 @@ sequelize.authenticate()
 
 
 let db = {}
+let modelNames = []
 
+// Get all the database table files from the files in this directory
 fs.readdirSync(__dirname)
 	.filter(file => {
 		return (file.indexOf('.') !== 0) && (file !=='index.js')
 	})
 	.forEach(file => {
 		let model = sequelize.import(path.join(__dirname, file))
 		db[model.name] = model
+		model.sync({force: false})
+		modelNames.push(model.name)
 	})
 
+
+modelNames.forEach(modelName => {
+	if (db[modelName].associate) { // if this function exists
+		db[modelName].associate(db) // call it
+		db[modelName].sync({alter: true}) // and re-sync the table
```

This is a key line. I don't want to do `force: true` since that will drop the tables each time the server is started, but I also need to edit the tables after the fact so that they all exist and the associations are known to be valid. This solution produces errors whenever the server is started and the tables already have the correct schema, but it does work. I could suppress the errors, but that would also hide real errors, so I'd rather not. If someone has a suggestion for how to do this more cleanly, I'd be happy to integrate that.

---

The docs claim that `alter: true` is [not recommended for production use](http://docs.sequelizejs.com/class/lib/sequelize.js~Sequelize.html#instance-method-sync); perhaps we should do this only in development?  I don't see the schema changing that much once we're done.

I can take a deeper look at this once I'm able to hack on this next week.  Ideally, we would run migrations before syncing the DB, and thus avoid the need for `alter`ation since the schema would match the latest migration after startup.  For development, though, this flag makes sense to set.